examples/low-level-api/listOffers.ts

```diff
@@ -2,7 +2,7 @@ import Bluebird from "bluebird";
 import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
 import { props as yp, rest, utils } from "yajsapi";
-import { Subscription } from "yajsapi/dist/rest/market";
+import { Subscription } from "../../yajsapi/rest/market";
```

Comment:
hmm... doesn't it mean that we're depending on the source version here? I guess it would be better to use the installed npm package if possible...

Reply:
this kind of formatting has been corrected automatically by the `prettier` tool with default rules. This can be tweaked of course, but I would leave the default rules which are often used in the js community

Reply:
yes, you are right, i'll fix it.